drivers/can/can_shell.c

```diff
@@ -366,7 +366,15 @@ static int cmd_can_bitrate_set(const struct shell *sh, size_t argc, char **argv)
 			return -EINVAL;
 		}
 
-		timing.sjw = CAN_SJW_NO_CHANGE;
+		if (argc >= 5) {
+			timing.sjw = (uint16_t)strtoul(argv[4], &endptr, 10);
```

Comment:
Why not dedicated `shell_strtoul`?

Reply:
Sure, but the rest of the file uses `strtoul()`. I must admit I don't see much benefit from using `shell_strtoul()` over `strtoul()` here.

Reply:
@bbilas I will follow up with a PR for converting the entire file to use `shell_strtoul()`. Okay?

Reply:
Sure, I was just curious ;) I have no strong objections.